app/models/user.rb

```diff
@@ -21,6 +21,26 @@ def email_checker(email)
     email.match(/^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9\-]+\.[a-zA-Z0-9\-.]+$/)
   end
 
+  def full_name
+    first_name + " " + last_name
+  end
+
+  def complete
+    orders.select { |order| order.status == 'complete'}
+  end
+
+  def paid
+    orders.select { |order| order.status == 'paid'}
+  end
+
+  def cancelled
+    orders.select { |order| order.status == 'cancelled'}
```

Comment:
Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.